02Lesson/01Homework/index.html

```diff
@@ -123,22 +123,19 @@
         </tr>
       </table>
     </section>
-
   </main>
+  <footer>
+    <section id="principles">
+      <div>
+        <h2>Leslie's Core Principles</h2>
+        <ol>
+          <li>Inbox Zero</li>
+          <li>Play Nice</li>
+          <li>Think twice, cut once</li>
+          <li>Under-promise, over-deliver</li>
+        </ol>
+      </div>
+    </section>
+  </footer>
 </body>
-
-<footer>
-  <section id="principles">
-    <div>
-      <h2>Leslie's Core Principles</h2>
-      <ol>
-        <li>Inbox Zero</li>
-        <li>Play Nice</li>
-        <li>Think twice, cut once</li>
-        <li>Under-promise, over-deliver</li>
-      </ol>
-    </div>
-  </section>
-</footer>
-
 </html>
```

Comment:
You went above and beyond the homework specs. Love the responsiveness and firm grasp of tables. Well done.